closure-typing/calculator.js

```diff
@@ -1,5 +1,26 @@
-function calculator() {
-  // your code here
+function calculator(inputExp) {
+
+    let reDecimal = new RegExp(/\d+(\.\d{0,2})?/);
+
+    if (reDecimal.test(inputExp)) {
+		
+        let mathOperator = inputExp.match(/[\+\-\/\*\(]/)[0];
```

Comment:
What with left brace in RegExp here? Here you saying that `(` can be treated as operator for your calculator, be carefull ;) try https://regex101.com/, to check your regexp